src/main.tf

```diff
@@ -142,6 +142,13 @@ locals {
       webhooks         = try(local.webhooks["services"], {})
       repository_files = local.rust_default.files
```

Comment:
A space is missing here, can you run `terraform fmt` and push the changes? That should fix the failing check.

Reply:
I did but nothing happened

Reply:
cd into the src/ directory first?

Reply:
that worked! thank you.